framework/db/sqlite/QueryBuilder.php

```diff
@@ -137,20 +137,17 @@ public function resetSequence($tableName, $value = null)
         $db = $this->db;
         $table = $db->getTableSchema($tableName);
         if ($table !== null && $table->sequenceName !== null) {
+            $tableName = $db->quoteTableName($tableName);
             if ($value === null) {
-                $key = reset($table->primaryKey);
-                $tableName = $db->quoteTableName($tableName);
+                $key = $this->db->quoteColumnName(reset($table->primaryKey));
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX({$key}) FROM {$tableName}")->queryScalar();
```

Comment:
I can only suggest some things but Yii core members decide. I think that using one syntax that has no side effects (and efficiently differs with this) is exactly about a logic and a consistency. But it's okay. :)